python/hail/expr/aggregators.py

```diff
@@ -724,6 +727,8 @@ def filter(condition, expr):
         lambda_result = to_expr(
             condition(
                 construct_expr(Reference(uid), agg._type, agg._indices, agg._aggregations, agg._joins, agg._refs)))
+        if not lambda_result._aggregations.empty():
+            raise ExpressionException("aggregators may not be used inside lambda functions")
```

Comment:
I'm a bit confused by this. Is this OK?

```python
t.aggregate(agg.collect(agg.filter(agg.sum(t.VAL) >= 70, t.ID)
```

it's not really about the lambda

Reply:
that will throw a different error -- can't aggregate an already-aggregated thing. Maybe we don't need this, actually.